dev/ci/ci-vst.sh

```diff
@@ -8,10 +8,6 @@ VST_CI_DIR="${CI_BUILD_DIR}/VST"
 # opam install -j ${NJOBS} -y menhir
 git_checkout "${VST_CI_BRANCH}" "${VST_CI_GITURL}" "${VST_CI_DIR}"
 
-# HACK: from the upstream makefile:
-#
-# default_target: _CoqProject version.vo msl veric floyd progs
-#
-# We have to omit progs as otherwise we timeout on Travis; once we
-# move to Gitlab we will able to just use `make`
-( cd "${VST_CI_DIR}" && make IGNORECOQVERSION=true _CoqProject version.vo msl veric floyd )
+# We have to omit progs as otherwise we timeout on Travis; on Gitlab
+# we will able to just use `make`
+( cd "${VST_CI_DIR}" && make IGNORECOQVERSION=true -o progs )
```

Comment:
Please improve/keep the old documentation, for the casual reader it may be clear what is going on.

Reply:
It says "we omit progs", that's as clear as it gets.

Reply:
Something such as:
```
The VST default is as of today:
default_target: ....
Due to build time constraints on Travis we omit the `progs` target, etc...
```

Reply:
I think the comment is clear enough like this. However, it contains a typo: "we will *be* able to"...